docker/compose/demo/compose.yml

```diff
@@ -103,7 +103,7 @@ services:
   solr:
     container_name: "solr"
     hostname: "solr"
-    image: solr:9.4.1
+    image: solr:9.6.1
```

Comment:
You're getting Docker stuff. Good. Can you please also bump the version in .env in the root of the repo?

Reply:
Done FWIW: I missed it since it hadn't been updated from 9.3.0